### PR TITLE
Added definitions for `to-markdown`

### PR DESCRIPTION
**Pull request by SuperPaintman**
Added definitions for [to-markdown](https://www.npmjs.com/package/to-markdown)

### CODE REVIEW

File: types/to-markdown/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/to-markdown/tsconfig.json]: Make this `true`.

File: types/to-markdown/to-markdown-tests.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+'use strict';
```
Comment (andy-ms) [inline types/to-markdown/to-markdown-tests.ts]: Don't need this

File: types/to-markdown/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// Type definitions for Marked (v3.0.4)
```
Comment (andy-ms) [inline types/to-markdown/index.d.ts]: Don't you mean for to-markdown?

**Review by ghost — COMMENTED** [review]
Needs a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

**Pull request merged**